### ISSUE TITLE
Bug: New merge method does not have "ignore_strand" argument => merge=True throws error

### ISSUE THREAD

**Issue by sschmeier** [daler/gffutils#157]
The new merge method that is used on line 1416 (interface.py) in FeatureDB has no ignore_strand:

https://github.com/daler/gffutils/blob/4b5b28e610a435af359ab1c31271deea1bae4c47/gffutils/interface.py#L1416

**Comment by mdshw5** [comment]
I just ran into this, and plan too submit a PR to update the docs. The fix for me was to do something like:

```python
import gffutils.merge_criteria as mc
merged_exons = db.merge(exons, merge_criteria=(mc.strand, mc.overlap_any_inclusive))
```

I like the new interface better but it's not well documented yet, and I think there should have been a deprecation warning or something to clue users in to ta major method change.

**Comment by daler** [comment]
Now addressed in v0.11.
